app/javascript/src/lib/OWLanguageLinter.js

```diff
@@ -622,3 +623,17 @@ function findUndefinedSubroutines(content) {
     }
   }
 }
+
+function findTripleEquals(content) {
+  const stringRanges = findRangesOfStrings(content);
+
+  for (const match of matchAllOutsideRanges(stringRanges, content, /={3}/g)) {
```

Comment:
While ={3} looks cool, `===` is much more readable

Reply:
```suggestion
  for (const match of matchAllOutsideRanges(stringRanges, content, /===/g)) {
```